py/client/pydeephaven/session.py

```diff
@@ -99,23 +113,29 @@ class Session:
         is_alive (bool): check if the session is still alive (may refresh the session)
     """
 
-    def __init__(self, host: str = None,
-                 port: int = None,
-                 auth_type: str = "Anonymous",
-                 auth_token: str = "",
-                 never_timeout: bool = True,
-                 session_type: str = 'python',
-                 use_tls: bool = False,
-                 tls_root_certs: bytes = None,
-                 client_cert_chain: bytes = None,
-                 client_private_key: bytes = None,
-                 client_opts: List[Tuple[str, Union[int, str]]] = None,
-                 extra_headers: Dict[bytes, bytes] = None):
+    host: str
+    port: int
```

Comment:
Do you have concerns about users accidentally changing these, so @property would be better?  I can go either way.